src/test/java/uk/gov/hmcts/reform/sscs/service/PreviewDocumentServiceTest.java

```diff
@@ -1,42 +1,57 @@
 import java.util.List;
+import java.util.regex.Pattern;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.sscs.ccd.domain.AbstractDocument;
+import uk.gov.hmcts.reform.sscs.ccd.domain.AbstractDocumentDetails;
 import uk.gov.hmcts.reform.sscs.ccd.domain.Appeal;
 import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
 import uk.gov.hmcts.reform.sscs.ccd.domain.SscsDocument;
 import uk.gov.hmcts.reform.sscs.ccd.domain.SscsDocumentDetails;
 
+@RunWith(MockitoJUnitRunner.class)
 public class PreviewDocumentServiceTest {
 
+    @InjectMocks
     private PreviewDocumentService previewDocumentService;
     private SscsCaseData sscsCaseData;
 
     @Before
     public void setup() {
-        previewDocumentService = new PreviewDocumentService();
-        sscsCaseData = SscsCaseData.builder().ccdCaseId("ccdId")
-                .appeal(Appeal.builder().build())
-                .build();
+        List<SscsDocument> docs = new ArrayList<>(List.of(
+            SscsDocument.builder()
+                .value(SscsDocumentDetails.builder()
+                    .documentFileName("oldDraft.doc")
+                    .documentType(DRAFT_ADJOURNMENT_NOTICE.getValue())
+                    .build())
+                .build()
+        ));
+        sscsCaseData = SscsCaseData.builder()
+            .ccdCaseId("ccdId")
+            .appeal(Appeal.builder().build())
+            .sscsDocument(docs)
+            .build();
     }
 
+    @DisplayName("Given draft adjournment notice already exists on case, then overwrite existing draft")
     @Test
-    public void givenDraftFinalDecisionAlreadyExistsOnCase_thenOverwriteExistingDraft() {
-        SscsDocument doc = SscsDocument.builder().value(SscsDocumentDetails.builder().documentFileName("oldDraft.doc").documentType(DRAFT_ADJOURNMENT_NOTICE.getValue()).build()).build();
-        List<SscsDocument> docs = new ArrayList<>();
-        docs.add(doc);
-        sscsCaseData.setSscsDocument(docs);
-
+    public void testOverwriteDraft() {
         previewDocumentService.writePreviewDocumentToSscsDocument(sscsCaseData, DRAFT_ADJOURNMENT_NOTICE, sscsCaseData.getAdjournCasePreviewDocument());
 
-        assertEquals(1, sscsCaseData.getSscsDocument().size());
-        assertEquals((String.format("Draft Adjournment Notice generated on %s.pdf", LocalDate.now().format(DateTimeFormatter.ofPattern("dd-MM-YYYY")))), sscsCaseData.getSscsDocument().get(0).getValue().getDocumentFileName());
+        assertThat(sscsCaseData.getSscsDocument())
+            .hasSize(1)
+            .extracting(AbstractDocument::getValue)
+            .extracting(AbstractDocumentDetails::getDocumentFileName)
+            .allSatisfy(s -> assertThat(s).containsPattern(Pattern.compile("Draft Adjournment Notice generated on \\d{1,2}-\\d{1,2}-\\d{4}\\.pdf")));
```

Comment:
Is this test expected to return today's date for the pdf name? If so, it might be better to use the removed pattern matching on LocalDate().now(), as the new code could pass with any combination of numbers as long as they maintain a xx-xx-xxxx format.

Reply:
This change is copying a similar one made by @TeaWithLucas elsewhere. I would need to defer to him for the reasoning behind changing from specifically today's date to any date.

Reply:
Hmm, I think @DGordonV1 is right on this, it is using the date today, you can split this test into two asserts, one for checking "Draft Adjournment Notice generated on" and the other checking that todays date is in the string

Reply:
```suggestion
.allSatisfy(s -> {
assertThat(s).contains("Draft Adjournment Notice generated on ");
assertThat(s).contains(...);
});
```